functions/main.py

```diff
@@ -139,9 +139,9 @@ def process_user_tree(event: Event[DocumentSnapshot | None]) -> None:
 
 @on_document_created(
     document="users/{userId}/trees/{treeId}",
-    memory=MemoryOption.MB_512,
+    memory=MemoryOption.GB_1,
 )
-def process_user_tree(event: Event[DocumentSnapshot | None]) -> None:
```

Comment:
I think this was supposed to be the `process_user_tree` the other has the 100 mb limit.